### ISSUE TITLE
Add basic autonomous routine for scoring with preloaded note

### ISSUE THREAD

**Issue by alexemrie** [Pumpkin-Bots/2024_Swerve_KitBot#4]
**tldr;** At the start of every match robots are put into an autonomous mode. Add a sequence of commands to drive the robot to the scoring position and shoot the preloaded note.
- See [Game Manual](https://firstfrc.blob.core.windows.net/frc2024/Manual/2024GameManual.pdf)

![Image](https://github.com/user-attachments/assets/698e1ca4-ccba-4b36-90ec-89165ef456b0)

We'll start in front of the subwoofer and shoot in autonomous mode.

- See [Choosing an Autonomous Program](https://docs.wpilib.org/en/stable/docs/software/dashboards/smartdashboard/choosing-an-autonomous-program-from-smartdashboard.html#choosing-an-autonomous-program)
- See [Organizing Command-Based Robot Projects](https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#subclassing-command-groups)

Other...
- May be different depending on alliance color/start position...

PathPlanner resources...
- https://github.com/team-9143/frc-bot-2024
- https://github.com/mjansen4857/pathplanner
- https://pathplanner.dev/home.html
- https://www.chiefdelphi.com/t/introducing-pathplanner-2-0/397629
